aten/src/ATen/native/quantized/AffineQuantizer.h

```diff
@@ -8,37 +8,37 @@
 namespace at {
 namespace native {
 
-Tensor& quantize_tensor_per_tensor_affine(
+TORCH_API Tensor& quantize_tensor_per_tensor_affine(
```

Comment:
@jerryzh168 you might be a better reviewer here - do we want these API's to get public symbols?

Reply:
this is not a public function I think

Reply:
> this is not a public function I think, what does "getting public symbols" entail here?

it's not, but it should be if `privateuse1` user like to implement quantization functions like `torch.quantize_per_tensor`, so we can reuse this native common functions.